autoPlay.js

```diff
@@ -297,37 +297,16 @@ function useClusterBombIfRelevant() {
 		}
 		//Bombs away if spawner and 2+ other monsters
 		if (enemySpawnerExists && enemyCount >= 3) {
-			triggerAbility(ABILITIES.CLUSTER_BOMB);
+			triggerAbility(ability);
 		}
 	}
 }
+function useClusterBombIfRelevant() {
+	useOffensiveAbilityIfRelevant(ABILITIES.CLUSTER_BOMB);
+}
 
 function useNapalmIfRelevant() {
-	//Check if Napalm is purchased and cooled down
-	if (hasPurchasedAbility(ABILITIES.NAPALM)) {
-		if (isAbilityCoolingDown(ABILITIES.NAPALM)) {
-			return;
-		}
-		
-		//Check lane has monsters to burn
```

Comment:
While a generalized method would be helpful, you can't but out the logic in each of the methods.

Also, I think naming the method `useOffensiveAbilityIfAvailable` would be better, then just do the ability-specific logic before calling it, and you should be good.